covid19-dashboard/main.py

```diff
@@ -12,180 +12,117 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from flask import Flask, send_from_directory, make_response
+from collections import defaultdict
```

Comment:
This is still not rebased to head. You want to do:

git pull origin master --rebase

Reply:
fixed.